### PR TITLE
Add event location markers to event map

### PR DESCRIPTION
**Pull request by sstamoulas**
https://www.pivotaltracker.com/n/projects/742821/stories/67698608

### CODE REVIEW

File: features/step_definitions/events_steps.rb [hunk 1]
```diff
@@ -1,9 +1,11 @@
 Given (/^I fill in the new event page validly$/) do
   fill_in 'event_title', with: 'Hackathon'
   fill_in 'event_description', with: 'Great place to brain storm'
-
   fill_in 'event_start_date', with: '20/04/2038'
   fill_in 'event_end_date', with: '28/04/2038'
+  
+  select('Cats Are Us', from: 'event_organisation_id')
```
Comment (nisevi) [inline features/step_definitions/events_steps.rb]: nice! I didn't know how to do this :+1:

File: Gemfile [hunk 2]
```diff
@@ -2,7 +2,7 @@ source 'https://rubygems.org'
 
 ruby '2.4.2'
 gem 'rails', '~> 4.2', '>= 4.2.7.1'
-gem 'pg', '0.21.0'
+gem 'pg', '~> 0.20.0'
```
Comment (tansaku) [inline Gemfile]: is there a particular reason we need to downgrade the pg gem?

File: app/controllers/events_controller.rb [hunk 3]
```diff
@@ -17,10 +17,9 @@ def show
   end
 
   def index
-    respond_to do |format|
-      format.html {  @events = Event.upcoming(10) }
-      format.json {  @events = Event.where(start_date: params[:start]..params[:end]) }
-    end
+    @events = Event.upcoming(10)
+    @markers = BuildMarkersWithInfoWindow
+      .with(Event.build_by_coordinates(@events), self)
```
Comment (tansaku) [inline app/controllers/events_controller.rb]: I notice we lose the json format here - fair enough - it's not covered by any tests - not a feature requested from the client ...

File: app/controllers/events_controller.rb [hunk 4]
```diff
@@ -30,7 +29,8 @@ def event_success
   end
 
   def event_params
-    params.require(:event).permit(:title, :description, :start_date, :end_date)
+    params.require(:event).permit(:title, :description, :postal,
```
Comment (tansaku) [inline app/controllers/events_controller.rb]: are we actually using this `postal` field?

File: app/controllers/events_controller.rb (hunk 3 above)
Comment (sstamoulas) [reply app/controllers/events_controller.rb]: I updated the changes per your request. With the exception of this one. Judging by your comments, I wasn't sure it should be updated. Please advise. Thanks.

Comment (sstamoulas) [reply app/controllers/events_controller.rb]: actually, I just submitted the fix for this as well.

**Pull request merged**